highfidelity/hifi
# Interstitial page/redirect domain improvements

- Improvement for MS#[18755](https://highfidelity.manuscript.com/f/cases/18755/Update-Visual-Assets-Within-404-Redirect-Room)
- Implicit fix for MS#[18557](https://highfidelity.manuscript.com/f/cases/18557/interstitial-page-on-desktop-cannot-see-call-to-action-text-on-the-ground)

## TEST PLAN
- Launch Interface
- In the menu, open `Settings > Developer Menu`
- In the menu, open `Developer > Console...`
- Run `Window.interstitialModeEnabled = true;` to enable the interstitial page for loading
- `CTRL-R` to reload cache/scripts
- The interstitial page should show up when navigating between domains
- When there is an error connecting to a domain via connection timeout, not logged in as an authorized user, or with a protocol version mismatch, the interstitial page should disappear and the user should land in the 404 serverless domain with text showing the error that occurred and a "Try Again" or "Back" button to either navigate to the same domain again or to go back to the domain before the one attempted
### Notes 
- toggling `Window.interstitialModeEnabled` and hitting `CTRL-R` afterwards to reload caches/scripts should toggle the interstitial page appearing and landing in the 404 redirect serverless domain accordingly
